### PR TITLE
UI Element: Input Field

### PR DESCRIPTION
**Pull request by heathervv**
I figured it made more sense to PR every element rather than the whole interface all at once!

In this PR you'll find all the base setup as well as the input field.

Please don't close the branch! I'd like to keep all interface stuff in this branch and then just PR them over to master whenever necessary.

### CODE REVIEW

**Comment by skimke** [comment]
Hmm since we're using gulp to compile Sass, maybe we should still look into using it for Babel. I'll look into it as a backlog type task, for now let's keep going and not lose steam.

http://babeljs.io/docs/setup/#installation

File: _/scss/modules/_search.scss [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+.search--input {
+    font-size: 16px;
+    line-height: 1.5;
+
+    width: 100%;
+    padding: 8px 36px 8px 12px;
+
+    border: 1px solid #ddd;
+    &:focus {
```
Comment (skimke) [inline _/scss/modules/_search.scss:33]: Not crucial, more of a style thing - could we maybe put a line break before the pseudo element selectors? (line 6 and 33)

**Comment by skimke** [comment]
Looks great 🎉 Let's merge it. We'll backlog the Babel installation with gulp, and the comment is just style related.

**Comment by heathervv** [comment]
I'll add a trello card to add line breaks before pseudo elements!

**Pull request merged**